Projects/Text-Reader/index.html

```diff
@@ -3,10 +3,11 @@
   <head>
     <meta charset="UTF-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <title>Text Reader</title>
+    <title><strong><u>Text Reader</u></strong></title>
```

Comment:
why this <strong><u> present here in the name , it looks odd

Reply:
![image](https://github.com/user-attachments/assets/d6da2425-1c0b-4990-8b95-4e09985485c2)